src/library_makeDynCall.js

```diff
@@ -19,7 +19,7 @@ mergeInto(LibraryManager.library, {
       "i", // The first argument is the function pointer to call
       // in the rest of the argument list, one 64 bit integer is legalized into
       // two 32 bit integers.
-      sig.slice(1).replace("j", "ii"),
+      sig.slice(1).split("").map(x => x === "j" ? "ii" : x).join("")
```

Comment:
Perhaps this?
```suggestion
      sig.slice(1).replace(/j/g, "ii");
```
I don't feel strongly though.

Reply:
It's a bit clearer.

Reply:
Should be a comma, not a semicolon.